src/main/resources/assets/api-docs.yaml

```diff
@@ -47,6 +47,23 @@ paths:
           description: The json representation for all data use questions + answers
         default:
           description: Unexpected error
+    post:
+      summary: Validate Data Use Terms
+      description: Checks if the Data Use terms has the correct format and context
+      parameters:
+        - name: body
+          in: body
+          description: DataUseV3
+          required: true
+          schema:
+            $ref: '#/definitions/DataUse'
```

Comment:
Pretty sure this definition is for the old version of `DataUse` - I think you'll need a new version.